tests/devapps/XamarinDev/XamarinDev/App.xaml.cs

```diff
@@ -13,18 +13,18 @@ public partial class App : Application
 
         public static object RootViewController { get; set; }
 
-        public const string DefaultClientId = "4b0db8c2-9f26-4417-8bde-3f0e3656f8e0";
+        public const string DefaultClientId = "16dab2ba-145d-4b1b-8569-bf4b9aed4dc8";
```

Comment:
i've changed all these values w/my android pr to fix the webview issue. so we no longer have different redirect uris

Reply:
ok